workers/unity/Assets/Playground/Scripts/Worker/MobileClientWorkerConnector.cs

```diff
@@ -1,20 +1,47 @@
+using System;
 using Improbable.Gdk.Core;
 using Improbable.Gdk.Mobile;
+using Improbable.Worker.CInterop;
 using UnityEngine;
 
 namespace Playground
 {
-    public class MobileClientWorkerConnector : DefaultMobileWorkerConnector
+    public class MobileClientWorkerConnector : WorkerConnector, IMobileSettingsProvider
```

Comment:
what will happen to the `DefaultMobileWorkerConnector`?

Reply:
It will be removed (I may have not done that yet)